tests/crm/conftest.py

```diff
@@ -97,9 +97,13 @@ def crm_interface(duthosts, enum_rand_one_per_hwsku_frontend_hostname, tbinfo, e
     elif len(mg_facts["minigraph_portchannel_interfaces"]) >= 4:
         crm_intf1 = mg_facts["minigraph_portchannel_interfaces"][0]["attachto"]
         crm_intf2 = mg_facts["minigraph_portchannel_interfaces"][2]["attachto"]
-    else:
+    elif len(mg_facts["minigraph_interfaces"]) > 2:
         crm_intf1 = mg_facts["minigraph_interfaces"][0]["attachto"]
         crm_intf2 = mg_facts["minigraph_interfaces"][2]["attachto"]
+    else:
```

Comment:
@mannytaheri for the else part, can we have a logic  to get the first set of available ports ? I see a similar logic in this PR https://github.com/Azure/sonic-mgmt/pull/3776,

Reply:
@judyjoseph in PR #3776 it is checking for admin_state of interface to be up "if intf_status[a_port]['admin_state'] == 'up':"
Shouldn't we be checking for oper_state to be up?